pkg/plugin/plugin_test.go

```diff
@@ -291,6 +291,41 @@ func TestUninstall(t *testing.T) {
 	assert.NoFileExists(t, pluginDir)
 }
 
+func TestInformation(t *testing.T) {
+	pluginName := "test_plugin"
+
+	tempDir := t.TempDir()
+	pluginDir := filepath.Join(tempDir, ".trivy", "plugins", pluginName)
+
+	os.Setenv("XDG_DATA_HOME", tempDir)
+	defer os.Unsetenv("XDG_DATA_HOME")
+
+	// Create the test plugin directory
+	err := os.MkdirAll(pluginDir, os.ModePerm)
+	require.NoError(t, err)
+	defer func() { _ = os.RemoveAll(pluginDir) }()
```

Comment:
I don't think we need to remove this directory explicitly since `tempDir` created by `t.TempDir` will be removed automatically after the test.

Reply:
Removed the removal